Packages/manifest.json

```diff
@@ -7,8 +7,9 @@
     "com.unity.test-framework": "1.1.33",
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.6.5",
+    "com.unity.toolchain.macos-x86_64-linux-x86_64": "2.0.9",
```

Comment:
Is this package needed?

Reply:
Not sure, it's automatically being installed if you have Linux target on a Mac Unity installation. 
I think I can remove it safely.